docs/src/wrapper.tsx

```diff
@@ -16,7 +16,17 @@ export const WrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
 export const WrapRootElement: GatsbyBrowser['wrapRootElement'] = ({
   element,
 }) => (
-  <ThemeProvider theme={b2bTheme}>
```

Comment:
After this removal our components doesn't have the b2b-theme style like before. Do you did this in regard of my switch-theme functionality?

Reply:
No, I just switched to the docs theme :)

Reply:
The theme from our docs app shouldn't leak into the examples in the first place.